src/app/modules/main/chat_section/controller.nim

```diff
@@ -700,3 +700,5 @@ proc getContractAddressesForToken*(self: Controller, symbol: string): Table[int,
 proc getCommunityTokenList*(self: Controller): seq[CommunityTokenDto] =
   return self.communityTokensService.getCommunityTokens(self.getMySectionId())
 
+proc getCheckChannelpermissionResponses*(self: Controller, communityId: string, chatId: string) =
```

Comment:
```suggestion
proc getCheckChannelPermissionResponses*(self: Controller, communityId: string, chatId: string) =
```

Reply:
Still one typo left :rofl:

Reply:
TIL NIM is actually case_insensitive_ for everything but the first letter :exploding_head: 

https://nim-lang.org/docs/manual.html#lexical-analysis-identifier-equality

Reply:
oh my goodness

Reply:
> Which kinda sucks major league as the Qt signals/slots/properties ARE case sensitive

I did run into such a scenario once and ended up debugging for several hours until I found the issue